test/passing/string.ml.ref

```diff
@@ -9,10 +9,16 @@ let f = function
 let _ = "\010\xFFa\o123\n\\\u{12345}aa🐪🐪🐪🐪🐪\n"
 
 let _ =
-  "aaaaaaaaaaaaaaaaaaaaaaaaa\n	\
-   bbbbbbbbbbbbbbbbbbbbbbbbbb\n	\
-   cccccccccccccccccc\n	\
-   "
+  "aaaaaaaaaaaaaaaaaaaaaaaaa\n\
+   \tbbbbbbbbbbbbbbbbbbbbbbbbbb\n\
+   \tcccccccccccccccccc\n\
+   \t"
+
+let _ =
+  "aaaaaaaaaaaaaaaaaaaaaaaaa\n\
+   \ bbbbbbbbbbbbbbbbbbbbbbbbbb\n\
```

Comment:
@jberdine, I was unable to have the indentation correct. (one to the left)

Reply:
Did you also try changing the `0` to `-1` in the `pre_break` in `fmt_line` above `fmt_lines`?

Reply:
The issue was that in preserve mode, the string contains `\ `  and the code was checking for leading ws only.